src/pages/events/new/index.tsx

```diff
@@ -0,0 +1,257 @@
+        start_time: startTime,
+        end_time: endTime,
+        event_type: eventType,
+        location: location,
+      })
+      if (error) {
+        toast.error("Something went wrong trying to add the event, please try again.")
+      } else {
+        toast.success("Your event has been added.")
+        router.push(`/events/${data.id}`)
+      }
+    } catch (error) {
+      toast.error("Something went wrong trying to add the event, please try again.")
+    }
+    setShowSpinner(false)
+  }
+
+  const isSelected = (field: string | number) => {
+    return field !== undefined && field != 0
+  }
+
+  const customDateInputField = (
+    <div className={"input-container"}>
+      <label htmlFor="eventDate" className="sr-only">
+        Pick an event date
+      </label>
+      {errors.date && <ErrorMessage dataCy={"input-date-error"} message={errors.date.message} />}
+
+      <div className="input">
+        <input
+          data-cy={"input-event-date"}
+          type="text"
+          value={format(new Date(eventDate), "cccc, LLLL d")}
+          {...register("date", {
+            required: "Date is required",
+            validate: () => {
+              const currentDate = new Date()
+              currentDate.setDate(currentDate.getDate() - 1) //substract one day to include today
+              return eventDate > currentDate || "The date has to be in the future"
+            },
+          })}
+        />
+        <span>
+          <CalendarIcon />
+        </span>
+      </div>
+    </div>
+  )
+
+  return (
+    <div className={"page-wrapper lg:w-3/5"}>
+      <div className={"flex justify-between"}>
+        <div className={"page-header"}>New Event</div>
+        <XMarkIcon
+          data-cy={"button-discard-new-suggestion"}
+          className={"h-8 w-8 cursor-pointer text-zinc-400 hover:text-red-500"}
+          onClick={() => router.push("/events")}
+        />
+      </div>
+      {showSpinner ? (
+        <div className={"h-[75vh] text-center"} data-cy="song-list-spinner">
+          <Spinner size={10} />
+        </div>
+      ) : (
+        <form className={"flex flex-col"} onSubmit={handleSubmit(submitNewEvent)}>
+          <div className={"input-container"}>
+            <label htmlFor="eventType" className="sr-only">
+              Enter the event type
+            </label>
+            {errors.eventType && (
+              <ErrorMessage dataCy={"input_eventtype_error"} message={errors.eventType.message} />
+            )}
+
+            <div className={`input w-full ${!isSelected(watch("eventType")) && "text-gray-400"}`}>
+              <select
+                data-cy={"event-type-select"}
+                {...register("eventType", {
+                  required: "Required",
+                  validate: (value) => isSelected(value) || "The event type needs to be selected",
+                })}
+              >
+                <option value="0" disabled selected hidden>
```

Comment:
```suggestion
              <select
                data-cy={"event-type-select"}
                defaultValue="0"
                {...register("eventType", {
                  required: "Required",
                  validate: (value) => isSelected(value) || "The event type needs to be selected",
                })}
              >
                <option value="0" disabled hidden>
```

To fix this error, remove selected & set the defaultValue in the select. Also do this for start + end time
![image](https://github.com/ASVGay/the-rhapsodies/assets/54884540/6d3f5021-36a2-445d-8f5f-fe71e93dea64)